recipe/conda_build_config.yaml

```
@@ -0,0 +1,3 @@
+channel_targets:
+  - conda-forge rdkit_rc
+  - conda-forge rdkit-dev_rc
```

```suggestion
channel_targets:
  - conda-forge rdkit_rc
```

I think this can only be one channel. I'd go for the first one!

---

Done. I was confused by the combination of the channel (?) and the package name